packages/core/modules-sdk/src/loaders/module-provider-loader.ts

```diff
@@ -43,7 +43,7 @@ export async function loadModuleProvider(
     }
   } catch (error) {
     throw new Error(
-      `Unable to find plugin ${pluginName} -- perhaps you need to install its package?`
+      `Unable to find module ${pluginName} -- perhaps you need to install its package?`
```

Comment:
If you feel like it, maybe let's rename the variable as well :)

Reply:
good point, I just used the github editor to do this one so didn't wanna mess up anything 😄